src/assessments/seo/largestKeywordDistanceAssessment.js

```diff
@@ -0,0 +1,133 @@
+const AssessmentResult = require( "../../values/AssessmentResult.js" );
+const Assessment = require( "../../assessment.js" );
+const merge = require( "lodash/merge" );
+const countWords = require( "../../stringProcessing/countWords.js" );
+const matchWords = require( "../../stringProcessing/matchTextWithWord.js" );
+const Mark = require( "../../values/Mark.js" );
+const marker = require( "../../markers/addMark.js" );
+const inRangeStartEndInclusive = require( "../../helpers/inRange.js" ).inRangeStartEndInclusive;
+
+/**
+ * Returns a score based on the largest percentage of text in
+ * which no keyword occurs.
+ */
+class largestKeywordDistanceAssessment extends Assessment {
+	/**
+	 * Sets the identifier and the config.
+	 *
+	 * @param {Object} config The configuration to use.
+	 *
+	 * @returns {void}
+	 */
+	constructor( config = {} ) {
+		super();
+
+		const defaultConfig = {
+			overRecommendedMaximumKeywordDistance: 40,
+			recommendedMaximumKeywordDistance: 30,
+			scores: {
+				good: 9,
+				okay: 6,
+				bad: 1,
+			},
+		};
+
+		this.identifier = "largestKeywordDistance";
+		this._config = merge( defaultConfig, config );
+	}
+
+	/**
+	 * Runs the largestKeywordDistance research and based on this returns an assessment result.
+	 *
+	 * @param {Paper} paper The paper to use for the assessment.
```

Comment:
I noticed these parameters aren't being aligned whereas this _is_ being done in `src/researches/largestKeywordDistance.js` (for the most part). Would be nice to keep this consistent.